ambry-protocol/src/test/java/com.github.ambry.protocol/RequestResponseTest.java

```diff
@@ -272,6 +272,7 @@ public void putRequestResponseTest() throws IOException {
 
   @Test
```

Comment:
Any specific test changes for the new version of `ReplicaMetadataResponse`?

Reply:
There weren't any before based on version but I added one to be safe